playwright/e2e/manage-account-avatar.spec.ts

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable playwright/no-skipped-test */
+import { test, expect } from '@playwright/test';
+import path from 'node:path';
+
+import { selectWallet, setCookieConsent } from '../utils/common.ts';
+
+test.describe('Avatar Uploader on Manage Account page', () => {
+  test.beforeEach(async ({ page, baseURL, context }) => {
+    await setCookieConsent(context, baseURL);
+    await page.goto('/');
+    await selectWallet(page);
+    await page.getByTestId('user-navigation-hamburger').click();
+    await page.getByTestId('user-menu').waitFor({ state: 'visible' });
+    await page
+      .getByTestId('user-menu')
+      .getByRole('link', { name: /manage account/i })
+      .click();
+    await page.waitForURL('/account/profile');
+  });
+
+  test('accepts valid PNG file with correct dimensions', async ({ page }) => {
+    const validPng = path.join(
+      __dirname,
+      '../fixtures/images/valid-avatar-300x300.png',
+    );
+
+    await page.getByRole('button', { name: 'Change avatar' }).click();
+    await page.locator('input[type="file"]').setInputFiles(validPng);
+
+    // Check success message
+    await expect(
+      page
+        .getByRole('alert')
+        .filter({ hasText: /profile image changed successfully/i }),
+    ).toBeVisible();
+  });
+
+  test('accepts valid JPG file with correct dimensions', async ({ page }) => {
+    const validJpg = path.join(
+      __dirname,
+      '../fixtures/images/valid-avatar-300x300.jpg',
+    );
+
+    await page.getByRole('button', { name: 'Change avatar' }).click();
+    await page.locator('input[type="file"]').setInputFiles(validJpg);
+
+    await expect(
+      page
+        .getByRole('alert')
+        .filter({ hasText: /profile image changed successfully/i }),
+    ).toBeVisible();
+  });
+
+  test('accepts valid SVG file with correct dimensions', async ({ page }) => {
+    const validSvg = path.join(
+      __dirname,
+      '../fixtures/images/valid-avatar-300x300.svg',
+    );
+
+    await page.getByRole('button', { name: 'Change avatar' }).click();
+    await page.locator('input[type="file"]').setInputFiles(validSvg);
+
+    await expect(
+      page
+        .getByRole('alert')
+        .filter({ hasText: /profile image changed successfully/i }),
+    ).toBeVisible();
+  });
+  // NOTE: enable this test once we have fixed the issue with the avatar uploader
```

Comment:
Has an issue been created for this? I tried searching and couldn't find one. Please leave a note on the issue to re-enable this test as part of that issue.